libraries/botbuilder-dialogs-adaptive/package.json

```diff
@@ -31,6 +31,7 @@
     "@types/node-fetch": "^2.5.3",
     "botbuilder-core": "~4.1.6",
     "botbuilder-dialogs": "~4.1.6",
+    "botbuilder-dialogs-declarative": "~4.1.6",
     "botbuilder-lg": "~4.1.6",
```

Comment:
I'm not sure we want to take this dependency as it means you can't use adaptive within a browser. I may file a DCR to refactor the declarative library some.